modules/react/layout/stories/examples/Grid/Basic.tsx

```diff
@@ -42,10 +42,9 @@ const Footer = ({children, ...props}: GridProps) => (
   </Grid>
 );
 
-export const UIExample = () => {
+export const Basic = () => {
   const parentCont = {
-    gridTemplateAreas:
-      "'Header Header Header Header' 'SideBar BodyContent BodyContent BodyContent' 'Footer Footer Footer Footer'",
+    gridTemplateAreas: "'Header Header' 'SideBar BodyContent' 'Footer Footer'",
```

Comment:
This shortened `gridTemplateAreas` value appears to work just as well (unless I'm missing something).

Reply:
That works for me!